runtime/ftplugin/help.lua

```diff
@@ -31,4 +31,5 @@ vim.keymap.set('n', 'gO', function()
   require('vim.vimhelp').show_toc()
 end, { buffer = 0, silent = true })
 
+vim.b.undo_ftplugin = vim.b.undo_ftplugin .. ' | call v:lua.vim.treesitter.stop()'
```

Comment:
Using `:call` with `v:lua` is better here as it accepts `|` as command separator.